pipeline/alert.go

```diff
@@ -209,6 +206,11 @@ type AlertNode struct {
 	// An empty value indicates the level is invalid and is skipped.
 	Crit *ast.LambdaNode
 
+	// Filter expression for reseting the WARNING alert level to OK.
+	Warn_reset *ast.LambdaNode
```

Comment:
Again can we name these WarnReset etc? Which in TICKscript will read `.warnReset`.